### PR TITLE
fix gc log analysis to only be present on gc logs

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
Change-Id: I9c2b9fcb09c9ca57a95971435119f1b92d0fa91f

### CODE REVIEW

**Comment by MatthewKhouzam** [comment]
oops

File: tracetypes/org.eclipse.tracecompass.incubator.jifa.core/src/org/eclipse/tracecompass/incubator/internal/jifa/core/gclog/xmlanalysis/gc_analysis.xml [hunk 1]
```diff
@@ -9,9 +9,11 @@
 		EPL-2.0 *************************************************************************** -->
 	<pattern version="1" id="system.gc.duration">
 		<head>
+			<traceType id="org.eclipse.tracecompass.jifa.gclog" />
 			<label value="GC Segments" />
 		</head>
 		<patternHandler>
+
```
Comment (bhufmann) [inline tracetypes/org.eclipse.tracecompass.incubator.jifa.core/src/org/eclipse/tracecompass/incubator/internal/jifa/core/gclog/xmlanalysis/gc_analysis.xml:16]: unnecessary change

**Review by bhufmann — APPROVED** [review]
Small comment (empty row). Up to you if you want to fix it.

**Review by bhufmann — APPROVED** [review]

**Comment by MatthewKhouzam** [comment]
Thanks
